stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/GsonChatApi.kt

```diff
@@ -621,7 +620,7 @@ internal class GsonChatApi(
         channelType: String,
         channelId: String,
         shadow: Boolean,
-    ): Call<CompletableResponse> {
+    ): Call<Unit> {
```

Comment:
Why not to change the signature of `RetrofitApi::banUser`?

Reply:
The Retrofit interfaces return models that represent the raw API responses, that's fine as-is. They just shouldn't be leaked further out, outside of these `ChatApi` implementations, because they're internal details of each `ChatApi`.

In this case, I had to replace it with `Unit` on the public interface because `MoshiChatApi` uses a different `CompletableResponse` class internally.